packages/react-apollo/mixin.core.js

```diff
@@ -1,8 +1,33 @@
 const { existsSync } = require('fs');
 const { Mixin } = require('hops-mixin');
+const { join } = require('path');
 const strip = require('strip-indent');
+const deprecate = require('depd')('hops-react-apollo');
+const { getApolloVersion } = require('./lib/apollo-version');
 
 class GraphQLMixin extends Mixin {
+  constructor(config, ...args) {
+    super(config, ...args);
+
+    // TODO: remove with Hops v15
+    this.apolloVersion = getApolloVersion();
+
+    if (this.apolloVersion === 2) {
+      deprecate(
+        '[DEP0006] Apollo v2 support in Hops has been deprecated and will be removed with Hops v15. Please upgrade to Apollo v3.'
+      );
+    } else {
+      if (typeof this.getLogger === 'function') {
+        this.getLogger().info(
```

Comment:
Not entirely sure, I suppose the `hops-info` package will always be available. But let's tackle that in a different change.

Reply:
I meant this specific log message. It shouldn't be news to Hops users doing it right.

Reply:
We thought it would be a good idea to include this message, to avoid confusion in case some teams have installed `@apollo/client` in their `package.json` but are actually still on v2.

Reply:
Okay, might make sense to handle that scenario. Although I think there's implicit knowledge required here. The reasoning `@apollo/client present => must be version 3` can be confusing otherwise. Then it's not helpful.

Reply:
Ok, we could implement a doctor check instead that breaks the build when `react-apollo` **and** `@apollo/client` are installed and drop this message?

Reply:
That sounds nice.

Reply:
Done